src/internalHelpers/_hslToHex.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import reduceHexValue from './_reduceHexValue'
+import toHex from './_numberToHex'
+
+function hslToHex(hue: number, saturation: number, lightness: number): string {
+  if (saturation === 0) {
+    // achromatic
+    const colorComponent = lightness * 255
+    return reduceHexValue(`#${toHex(colorComponent)}${toHex(colorComponent)}${toHex(colorComponent)}`)
+  }
+
+  // formular from https://en.wikipedia.org/wiki/HSL_and_HSV
+  const huePrime = (hue % 360) / 60
+  const chroma = (1 - Math.abs((2 * lightness) - 1)) * saturation
+  const secondComponent = chroma * (1 - Math.abs((huePrime % 2) - 1))
+
+  let red
+  let green
+  let blue
+
+  if (huePrime >= 0 && huePrime < 1) {
+    red = chroma
+    green = secondComponent
+    blue = 0
+  } else if (huePrime >= 1 && huePrime < 2) {
+    red = secondComponent
+    green = chroma
+    blue = 0
+  } else if (huePrime >= 2 && huePrime < 3) {
+    red = 0
+    green = chroma
+    blue = secondComponent
+  } else if (huePrime >= 3 && huePrime < 4) {
+    red = 0
+    green = secondComponent
+    blue = chroma
+  } else if (huePrime >= 4 && huePrime < 5) {
+    red = secondComponent
+    green = 0
+    blue = chroma
+  } else if (huePrime >= 5 && huePrime < 6) {
+    red = chroma
+    green = 0
+    blue = secondComponent
+  }
+
+  const lightnessModification = lightness - (chroma / 2)
+  const addLightAndConvert = (color) => toHex(Math.round((color + lightnessModification) * 255))
```

Comment:
Good point, I just extracted it. I think it will only be used in hslToHex as it's specific to the hslToRgb formula, but once I see us using it twice I will extract it 👍